src/anim/binder/default-anim-binder.js

```diff
@@ -82,20 +82,21 @@ class DefaultAnimBinder {
             'weights': function (node) {
                 var meshInstances = findMeshInstances(node);
                 if (meshInstances) {
-                    var morphInstance;
+                    var morphInstances = [];
                     for (var i = 0; i < meshInstances.length; ++i) {
                         if (meshInstances[i].node.name === node.name && meshInstances[i].morphInstance) {
```

Comment:
is this part needed, considering we find mesh instances from that node?
`meshInstances[i].node.name === node.name`

Reply:
maybe that's for the model component actually.

Reply:
Yep this is used for the model component.